nodeadm/internal/containerd/config.go

```diff
@@ -32,6 +37,27 @@ type containerdTemplateVars struct {
 	RuntimeBinaryName string
 }
 
+func generateContainerdConfigTemplate() (*template.Template, error) {
```

Comment:
nit: i don't feel like any "generation" is happening
```suggestion
func getContainerdConfigTemplate() (*template.Template, error) {
```